go/vt/vtgate/evalengine/integration/fuzz_test.go

```diff
@@ -381,11 +381,23 @@ func compareResult(localErr, remoteErr error, localVal, remoteVal sqltypes.Value
 		if err != nil {
 			return fmt.Sprintf("error converting remote value to datetime: %v", err)
 		}
-		if !closeDatetime(localDatetime, remoteDatetime, 100*time.Millisecond) {
+		if !closeDatetime(localDatetime, remoteDatetime, 1*time.Second) {
+			return fmt.Sprintf("different results: %s; mysql response: %s (local collation: %s; mysql collation: %s)",
+				localVal.String(), remoteVal.String(), localCollationName, remoteCollationName)
```

Comment:
I see that all the `return`s in this function are returning an error as a string, wouldn't it be better to change the return type from `string` to `error` and use `fmt.Errorf` instead of `fmt.Sprintf`?

Reply:
Yeah that seems like a sensible change.